actions/docs-verifier/src/ActionRunner/ActionRunner.csproj

```diff
@@ -2,12 +2,19 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\MarkdownLinksVerifier\MarkdownLinksVerifier.csproj" />
     <ProjectReference Include="..\RedirectionVerifier\RedirectionVerifier.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.0">
```

Comment:
Why do you need to add this reference?

Reply:
Good question, I believe Visual Studio added this on its own. I didn't explicitly add it, but it certainly doesn't hurt anything. I can remove it if you'd like.

Reply:
This should be removed and the version should be updated in https://github.com/dotnet/docs-tools/blob/0c7914887bd21500d2dd4d1994c26ce982f3a237/actions/docs-verifier/Directory.Build.props#L15

Reply:
It shouldn't be needed even if you are running code analysis. The analyzers are part of the .NET SDK now.

Reply:
Ah, I might have added it to workaround some bug that wasn't yet shipped to analyzers in the .NET SDK.